cmd/kola/options.go

```diff
@@ -64,6 +66,7 @@ func init() {
 	sv(&outputDir, "output-dir", "", "Temporary output directory for test data and logs")
 	sv(&kola.TorcxManifestFile, "torcx-manifest", "", "Path to a torcx manifest that should be made available to tests")
 	root.PersistentFlags().StringVarP(&kolaPlatform, "platform", "p", "qemu", "VM platform: "+strings.Join(kolaPlatforms, ", "))
+	root.PersistentFlags().StringVarP(&kolaChannel, "channel", "c", "edge", "Channel: "+strings.Join(kolaChannels, ", "))
```

Comment:
I'm not sure if having `edge` as default makes sense. I think we would set it explicitly anyway and for users `stable` would be more common, or?

Reply:
Are stable users running kola tests?

Reply:
+1 on making `stable` by default.

Reply:
Agreed @pothos, the channels have to be explicitly mentioned while  we are  our running the tests. I've updated the default to `stable`.